go.mod

```diff
@@ -58,13 +58,13 @@ require (
 	github.com/openshift/imagebuilder v1.2.19
 	github.com/rootless-containers/rootlesskit/v2 v2.3.5
 	github.com/shirou/gopsutil/v4 v4.25.10
-	github.com/sirupsen/logrus v1.9.3
+	github.com/sirupsen/logrus v1.9.4-0.20230606125235-dd1b4c2e81af
 	github.com/spf13/cobra v1.10.1
 	github.com/spf13/pflag v1.0.10
 	github.com/stretchr/testify v1.11.1
 	github.com/vbauerster/mpb/v8 v8.11.2
 	github.com/vishvananda/netlink v1.3.1
-	go.podman.io/common v0.66.1-0.20251122023359-22d50c57903c
+	go.podman.io/common v0.66.1-0.20251125102557-af8a8926b275
```

Comment:
please update all go.podman.io modules at once as they ar epart of the sma erepo they use the same commits and should only be used together